docs/introduction/Base-App.rst

```diff
@@ -421,6 +422,19 @@ Each pair on the passed list represents a path to be disabled (the key is the ba
 		.annotatedWith(Names.named("RejectedApiPaths"))
 		.toInstance(...); 
 
+- Sidechain Application stopper
+It is a customized class instance which implements the public interface 'SidechainAppStopper' and must provide an
+implementation of the method 'void stopAll()'. Such a method is called by the SDK when the node stop procedure is initiated.
+Such a procedure can be explicitly triggered via the API 'node/stop' or can be triggered when the JVM is shutting down,
+for instance when a SIGINT is received.
+In the custom implementation for instance, custom storages should be closed or any resources should be properly released.
+An example is provided in the "SimpleApp" with the SimpleAPpStopper.java class.
```

Comment:
is "SimpleAPpStopper.java" a typo ? Should it be "SimpleAppStopper" ?

Reply:
Addressed here: https://github.com/HorizenOfficial/developer-horizen-global/pull/61
pls, approve and merge both PRs